### ISSUE TITLE
Some videos not hiding despite being watched

### ISSUE THREAD

**Issue by Coalbus** [OsaSoft/youtube-better-subscriptions#87]
First of all, thank you for your work on this extension. It makes Youtube's interface much less frustrating and cluttered with videos I've already seen. This is the only extension I've found for this use that is actually good.

I've noticed after the recent update that some videos, despite being watched and showing the red watch progress bar, are not being hidden. Some are, however. If I show watched videos, the ones that are being hidden appear sort of greyed out. 

In the screenshot below, the green square is the video that was being hidden properly. You'll notice that it appears greyed out. The purple square is around videos that have been watched but yet are not being hidden after being watched, despite showing the red progress bar.

![image](https://user-images.githubusercontent.com/30502962/103152660-cbac7e80-474f-11eb-9491-6bccba0d49a2.png)

I don't know if this is a bug, or intention behavior. If this is intentional, perhaps because it didn't register the video as being fully watched, I would much prefer the videos be hidden after ANY amount of watch time (as it did before). I don't always watch to the end of the video due to the end being mainly post-amble or sponsor segments.

**Comment by OsaSoft** [comment]
Hi and thanks for the feedback, both here and on the other issue.
This is in fact intentional behaviour introduced in 0.14, as many people were complaining about the unreliability of using YT's watched status/red bar to mark videos.

I personally also liked this feature, as it automatically hid videos that I watched eg on my phone. This is why I decided the next thing Im doing is bringing it back as an optional feature (configurable in settings), with the user able to select a percentage which marks it as watched. This will most likely be v0.15, which Id very much like to push before the end of the year
